packages/cart/src/repositories/line-item.ts

```diff
@@ -0,0 +1,24 @@
+import { Context } from "@medusajs/types"
+import { DALUtils } from "@medusajs/utils"
+import { SqlEntityManager } from "@mikro-orm/postgresql"
+import { LineItem } from "@models"
+import { UpdateLineItemDTO } from "../types"
+
+export class LineItemRepository extends DALUtils.mikroOrmBaseRepositoryFactory<LineItem>(
+  LineItem
+) {
+  async update(
+    data: { lineItem: LineItem; update: UpdateLineItemDTO }[],
+    context: Context = {}
+  ): Promise<LineItem[]> {
+    const manager = this.getActiveManager<SqlEntityManager>(context)
+
+    const entities = data.map(({ lineItem, update }) => {
+      return manager.assign(lineItem, update)
+    })
+
+    manager.persist(entities)
+
+    return entities
```

Comment:
Yes, good push. Right now, I would allow all standard props to be updated on a CartLineItem. Any specific things you have in mind for the controlled update that could lead to unexpected behavior?

Reply:
Yes, the collection works a bit differently with mikro orm. Thats why i am suggesting that but at the same time it would require some more testing/serializing etc. We can see that another time if you prefer

Reply:
I am working on the adjustments and tax lines in a separate PR (yet to be opened), so I'd like to keep that separate for now. This service method will certainly change as these concepts are introduced.